### PR TITLE
chore(testapp): upgrade to angular 2.0.0-rc.7

### PR DESCRIPTION
**Pull request by cnishina**
- remove hybrid test app test

### CODE REVIEW

**Comment by juliemr** [comment]
Travis failure seems legitimate.

File: scripts/test.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ var passingTests = [
   'node built/cli.js spec/customFramework.js',
   'node built/cli.js spec/noGlobalsConf.js',
   'node built/cli.js spec/angular2Conf.js',
-  'node built/cli.js spec/hybridConf.js',
+  // 'node built/cli.js spec/hybridConf.js',
```
Comment (juliemr) [inline scripts/test.js]: Please add a TODO to reenable this, and note why we had to comment it out.

File: testapp/package.json [hunk 2]
```diff
@@ -10,27 +10,28 @@
...
   "devDependencies": {
-    "concurrently": "2.0.0",
-    "lite-server": "2.1.0",
-    "typescript": "1.8.2",
-    "typings":"0.6.8",
+    "concurrently": "2.2.0",
+    "lite-server": "2.2.0",
+    "typescript": "1.8.10",
+    "typings":"1.0.4",
+    "canonical-path": "0.0.2",
```
Comment (juliemr) [inline testapp/package.json]: canonical-path appears to be unused

File: testapp/package.json [hunk 3]
```diff
@@ -10,27 +10,28 @@
...
+    "@angular/platform-browser-dynamic": "2.0.0-rc.7",
+    "@angular/router": "3.0.0-rc.3",
+    "@angular/upgrade": "2.0.0-rc.7",
+    "systemjs": "0.19.27",
+    "core-js": "2.4.1",
+    "reflect-metadata": "0.1.3",
+    "rxjs": "5.0.0-beta.12",
+    "zone.js": "0.6.21",
+    "angular2-in-memory-web-api": "0.0.19",
+    "bootstrap": "^3.3.6"
```
Comment (juliemr) [inline testapp/package.json]: Pretty sure we don't need bootstrap

Comment (cnishina) [inline testapp/package.json]: Removed. I believe this is residual from trying to update the hybrid.

File: testapp/package.json (hunk 2 above)
Comment (cnishina) [inline testapp/package.json]: Removed. I believe this is residual from trying to update the hybrid.

File: scripts/test.js (hunk 1 above)
Comment (cnishina) [inline scripts/test.js]: Added a TODO

**Comment by cnishina** [comment]
Agree, Travis failures are issues that needs investigation.

**Comment by cnishina** [comment]
After investigation, navigating to `ng2/#/async` would resolve to the HomeComponent. Router location strategy required `useHash` to be set to true.

File: spec/noGlobals/noGlobals_spec.js [hunk 4]
```diff
@@ -21,10 +21,13 @@ describe('configuration with no globals', function() {
   });
 
   it('should be able to use methods under the protractor namespace', function() {
+    console.log(URL);
```
Comment (juliemr) [inline spec/noGlobals/noGlobals_spec.js]: Remove logging.

**Comment by cnishina** [comment]
Suite passes locally. Travis w/ node 6 was experiencing flakes. Retrying the test.

**Comment by juliemr** [comment]
LGTM after comment-removal and Travis is happy.

File: spec/noGlobals/noGlobals_spec.js (hunk 4 above)
Comment (cnishina) [inline spec/noGlobals/noGlobals_spec.js]: 😞 Fixing.

**Pull request merged**